AFQ/api.py

```diff
@@ -12,6 +12,28 @@
 def do_preprocessing():
     raise NotImplementedError
 
+# 
+# # Set the default set as a module-wide constant:
+# BUNDLE_NAMES = ["ATR", "CGC", "CST",
+#                # "FA", "FP",
+#                "HCC", "IFO", "ILF",
+#                "SLF", "ARC", "UNC"]
+#
+# # Read in the standard templates:
+# AFQ_TEMPLATES = afd.read_templates()
+# # For the arcuate, we need to rename a few of these and duplicate the SLF ROI:
+# AFQ_TEMPLATES['ARC_roi1_L'] = AFQ_TEMPLATES['SLF_roi1_L']
+# AFQ_TEMPLATES['ARC_roi1_R'] = AFQ_TEMPLATES['SLF_roi1_R']
+# AFQ_TEMPLATES['ARC_roi2_L'] = AFQ_TEMPLATES['SLFt_roi2_L']
+# AFQ_TEMPLATES['ARC_roi2_R'] = AFQ_TEMPLATES['SLFt_roi2_R']
+#
```

Comment:
I prefer to simply use git history to access old code, rather than adding comments. Also want to make sure that this was intentionally commented out when moved.